caluma/caluma_workflow/jexl.py

```diff
@@ -145,3 +188,48 @@ def extract_tasks(self, expr):
                 )
             )
         )
+
+    @classmethod
+    def get_all_registered_dynamic_tasks(cls):
+        return chain(
+            *[
+                dynamic_tasks_class().get_registered_dynamic_tasks()
+                for dynamic_tasks_class in cls.dynamic_tasks_classes
+            ]
+        )
+
+    def task_transform(self, task_name, case, user, prev_work_item, dynamic_context):
+        all_tasks = self.tasks_transform(
+            [task_name], case, user, prev_work_item, dynamic_context
+        )
+
+        if len(all_tasks) > 1:
+            raise ValidationError(
+                f"The dynamic task `{task_name}` used by the `task` transform should not return more than one task -- it returned {len(all_tasks)}"
```

Comment:
Long line - should be split for readability/pep8:
```suggestion
                f"The dynamic task `{task_name}` used by the `task` transform should "
                f"not return more than one task -- it returned {len(all_tasks)}"
```

Reply:
Fixed